src/views/SignUpPage/components/ShopInfo.tsx

```diff
@@ -1,26 +1,138 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { styled } from 'styled-components';
 
+import { IcSearch } from '../assets/icons';
+import { days } from '../constants/days';
+import { HELPER_MESSAGE } from '../constants/message';
 import { TOTAL_STEP } from '../constants/step';
 import { EnterProfileProp } from '../utils/enterProfileProp';
 
+import Field from './Field';
+
 import Button from '@/views/@common/components/Button';
+import Input from '@/views/@common/components/Input';
+import Modal from '@/views/@common/components/Modal';
 import ProgressBar from '@/views/@common/components/ProgressBar';
 
 const ShopInfo = ({ setStep }: EnterProfileProp) => {
+  //이동
+  const navigate = useNavigate();
+  const [isOpenModal, setOpenModal] = useState(false);
+
+  const [isClicked, setIsClicked] = useState<boolean[]>([false, false, false, false, false, false, false]);
```

Comment:
```suggestion
  const [isClicked, setIsClicked] = useState<boolean[]>(new Array(7).fill(false));
```
요렇게 배열 만들어 줄 수도 있습니당!